app/league/config_utils.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""Helper utility for managing application settings."""
+import json
+from os.path import exists, getsize, join
+
+from flask import current_app
+
+
+def update_config_file(new_config):
```

Comment:
Hmm, do you have any suggested reading / horror stories?

I can implement what you want, but I can't guarantee that I'll get to it before Monday.